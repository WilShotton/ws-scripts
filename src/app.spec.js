import Enzyme, { mount } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import React from 'react'
import App from './app'


Enzyme.configure({ adapter: new Adapter() })

describe('<App />', () => {

    it('should match snapshot', () => {

        const wrapper = mount(<App />)

        expect(wrapper).toMatchSnapshot()
    })

    it('should be true', () => {

        expect(true).toBe(true)
    })
})
