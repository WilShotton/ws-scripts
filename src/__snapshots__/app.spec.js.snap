// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should match snapshot 1`] = `
ReactWrapper {
  "length": 1,
  Symbol(enzyme.__unrendered__): <App />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {},
    "ref": null,
    "rendered": Object {
      "instance": <div>
        <p>
          Hello World!
        </p>
      </div>,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <p>
          Hello World!
        </p>,
        "className": undefined,
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": <p>
            Hello World!
          </p>,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Hello World!",
          },
          "ref": null,
          "rendered": Array [
            "Hello World!",
          ],
          "type": "p",
        },
      ],
      "type": "div",
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {},
      "ref": null,
      "rendered": Object {
        "instance": <div>
          <p>
            Hello World!
          </p>
        </div>,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <p>
            Hello World!
          </p>,
          "className": undefined,
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": <p>
              Hello World!
            </p>,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Hello World!",
            },
            "ref": null,
            "rendered": Array [
              "Hello World!",
            ],
            "type": "p",
          },
        ],
        "type": "div",
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
